minijinja/src/error.rs

```diff
@@ -272,13 +277,22 @@ impl Error {
         }
     }
 
-    /// Returns the line number where the error occurred.
+    /// Returns the span where the error occurred.
     #[cfg(feature = "debug")]
-    pub(crate) fn span(&self) -> Option<Span> {
+    pub fn span(&self) -> Option<Span> {
         self.repr.span
     }
 
-    /// Returns the template debug information is available.
+    /// Returns the template source if available.
+    #[cfg(feature = "debug")]
+    pub fn source(&self) -> Option<&str> {
```

Comment:
This should be `template_source` to not shadow the `source` method from `std::error::Error`.